muspinsim_plot/muspinsim_plot.xml

```diff
@@ -0,0 +1,105 @@
+                <option value="r">Red</option>
+                <option value="g">Green</option>
+                <option value="b">Blue</option>
+            </param>
+            <conditional name="series_type">
+                <param type="select" name="type" label="Series Type" >
+                    <option value="line">Line</option>
+                    <option value="points">Points</option>
+                </param>
+                <when value="line">
+                   <param type="select" name="linestyle" label="Line type">
+                       <option value="solid">Solid</option>
+                       <option value="dashed">Dashed</option>
+                       <option value="dotted">Dotted</option>
+                   </param>
+                   <param type="float" name="linewidth" label="Line width" value="1.0"/>
+                </when>
+                <when value="points">
+                    <param type="select" name="pointstyle" label="Point Type">
+                        <option value="s">Square</option>
+                        <option value="o">Circle</option>
+                    </param>
+                    <param type="float" name="pointscale" label="Point Scale" value="1.0"/>
+                </when>
+            </conditional>
+        </repeat>
+    </inputs>
+    <outputs>
+        <data format="pdf" name="out_file_pdf" from_work_dir="outfile.pdf" >
+            <filter>(outfile_type == 'pdf')</filter>
+        </data>
+        <data format="png" name="out_file_png" from_work_dir="outfile.png">
+            <filter>(outfile_type == 'png')</filter>
+        </data>
+    </outputs>
+    <tests>
+        <test>
+            <param name="title" value="Temperature Example"/>
+            <param name="xlab" value="Time"/>
+            <param name="ylab" value="Asymmetry"/>
+            <param name="outfile_type" value="png"/>
+            <param name="mu_data" value="high_T.dat" ftype="txt"/>
+            <param name="mu_label" value="High T limit"/>
+            <param name="type" value="points"/>
+            <param name="pointstyle" value="s"/>
+            <param name="colour" value="r"/>
+            <param name="pointscale" value="2.0"/>
+            <param name="mu_out_series_1|mu_data" value="low_T.dat" ftype="txt"/>
+            <param name="mu_out_series_1|mu_label" value="T=1 K"/>
+            <param name="mu_out_series_1|type" value="line"/>
+            <param name="mu_out_series_1|linestyle" value="solid"/>
+            <param name="mu_out_series_1|colour" value="k"/>
+            <param name="mu_out_series_1|linewidth" value="1.0"/>
+            <output name="out_file_png" file="temp_example.png" ftype="png" compare="sim_size" />
+        </test>
+        <test>
+            <param name="title" value="Fitting Example"/>
+            <param name="xlab" value="Time"/>
+            <param name="ylab" value="Asymmetry"/>
+            <param name="outftype" value="pdf"/>
```

Comment:
1. Should this name be `outfile_type`?
2. Related, I see both `out_file...` and `outfile...` used in parameter names - better to use only one style. I favour `out_file` personally, but up to you.

Reply:
Fixed issue, also refactored to use only `out_file..` parameter names [233c849](https://github.com/muon-spectroscopy-computational-project/muon-galaxy-tools/pull/55/commits/233c849cefab10d82533389d8480f2cdd863e868)